src/test/java/uk/gov/hmcts/reform/unspec/aspect/NoOngoingBusinessProcessAspectTest.java

```diff
@@ -43,76 +45,132 @@
 
     @Autowired
     NoOngoingBusinessProcessAspect aspect;
-    @MockBean
-    ProceedingJoinPoint proceedingJoinPoint;
-
-    @Test
-    @SneakyThrows
-    void shouldProceedToMethodInvocation_whenNoOngoingBusinessProcess() {
-        AboutToStartOrSubmitCallbackResponse response = AboutToStartOrSubmitCallbackResponse.builder().build();
-        when(proceedingJoinPoint.proceed()).thenReturn(response);
-
-        CallbackParams callbackParams = CallbackParamsBuilder.builder()
-            .type(ABOUT_TO_START)
-            .request(CallbackRequest.builder()
-                         .eventId(RESPOND_EXTENSION.name())
-                         .caseDetails(CaseDetailsBuilder.builder().atStateExtensionRequested().build())
-                         .build())
-            .build();
-        Object result = aspect.checkOngoingBusinessProcess(proceedingJoinPoint, callbackParams);
-
-        assertThat(result).isEqualTo(response);
-        verify(proceedingJoinPoint).proceed();
-    }
 
-    @SneakyThrows
-    @ParameterizedTest
-    @NullSource
-    @EnumSource(value = BusinessProcessStatus.class, names = "FINISHED", mode = EnumSource.Mode.INCLUDE)
-    void shouldProceedToMethodInvocation_whenBusinessProcessStatusIsNullOrFinished(BusinessProcessStatus status) {
-        AboutToStartOrSubmitCallbackResponse response = AboutToStartOrSubmitCallbackResponse.builder().build();
-        when(proceedingJoinPoint.proceed()).thenReturn(response);
-
-        CallbackParams callbackParams = CallbackParamsBuilder.builder()
-            .type(ABOUT_TO_START)
-            .request(CallbackRequest.builder()
-                         .eventId(CONFIRM_SERVICE.name())
-                         .caseDetails(CaseDetailsBuilder.builder().data(
-                             CaseDataBuilder.builder()
-                                 .atStateClaimCreated()
-                                 .businessProcess(BusinessProcess.builder().status(status).build())
-                                 .build()).build())
-                         .build())
-            .build();
-        Object result = aspect.checkOngoingBusinessProcess(proceedingJoinPoint, callbackParams);
-
-        assertThat(result).isEqualTo(response);
-        verify(proceedingJoinPoint).proceed();
+    @Nested
+    class NormalUserEvent {
```

Comment:
I will think to rename this.

Reply:
I would go with UserEvent and CamundaEvent